src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -23,6 +23,7 @@
 #include "phongo_bson_encode.h"
 #include "phongo_error.h"
 
+#include "BSON/Int64.h"
```

Comment:
Not quite. `php_phongo_bson_new_int64` was previously declared in `phongo_bson.h`. This file is included in `phongo_structs.h`, which again is included in `php_phongo.h`, which is included in almost all of our files. With the function being moved to Int64.h, the `_INT64` macros require the inclusion of this file. What makes the problem worse is that the `_INT64` macros pass through to the `_LONG` macros on 64-bit machines and only make use of `php_phongo_bson_new_int64` on 32-bit machines. This makes this hard to spot during development, as tooling such as CLion marks the include as unnecessary.

Thinking about this some more, it would make sense to add `Int64.h` to `phongo_compat.h` as that's where usage comes from. That's probably the better alternative.